diff --git a/node_modules/@cjblack/expo-audio-stream/ios/AudioUtils.swift b/node_modules/@cjblack/expo-audio-stream/ios/AudioUtils.swift
index f9559a8..eee0178 100644
--- a/node_modules/@cjblack/expo-audio-stream/ios/AudioUtils.swift
+++ b/node_modules/@cjblack/expo-audio-stream/ios/AudioUtils.swift
@@ -1,5 +1,6 @@
 import AVFoundation
 import ExpoModulesCore
+import Accelerate
 
 public enum SoundPlayerError: Error {
     case invalidBase64String
